src/tmpfiles/tmpfiles.c

```diff
@@ -2913,16 +3035,34 @@ static int parse_line(
 
         if (!empty_or_dash(age)) {
                 const char *a = age;
+                _cleanup_free_ char *seconds = NULL, *age_by = NULL;
 
                 if (*a == '~') {
                         i.keep_first_level = true;
                         a++;
                 }
 
-                r = parse_sec(a, &i.age);
+                /* Format: "age[:age-by]", where age-by is "[abcm]time" */
+                if (!strchr(a, ':')) {
+                        seconds = strdup(a);
+                } else {
+                        r = split_pair(a, ":", &seconds, &age_by);
+                        if (r < 0) {
+                                *invalid_config = true;
+                                return log_syntax(NULL, LOG_ERR, fname, line, r, "Failed to parse age '%s'.", age);
+                }
```

Comment:
there's a } gone missin here

Reply:
It's bad indentation. But yeah, I see that it can be confusing. Will fix.